.ci/install_firecracker.sh

```diff
@@ -26,17 +27,23 @@ if [ "$docker_version" != "18.06" ]; then
 	die "Firecracker hypervisor only works with docker 18.06"
 fi
 
-# This is the initial release of Kata
-# Containers that introduces support for
-# the Firecracker hypervisor
-release_version="1.5.0-rc2"
-file_name="kata-fc-static-${release_version}-${arch}.tar.gz"
-url="https://github.com/kata-containers/runtime/releases/download/${release_version}/${file_name}"
-echo "Get static binaries from release version ${release_version}"
-curl -OL ${url}
+# Get url for firecracker from runtime/versions.yaml
+firecracker_repo=$(get_version "assets.hypervisor.firecracker.url")
+[ -n "$firecracker_repo" ] || die "failed to get firecracker repo"
+firecracker_repo=${firecracker_repo/https:\/\//}
 
-echo "Decompress binaries from release version ${release_version}"
-sudo tar -xvf ${file_name} -C /
+# Get version for firecracker from runtime/versions.yaml
+firecracker_version=$(get_version "assets.hypervisor.firecracker.version")
+[ -n "$firecracker_version" ] || die "failed to get firecracker version"
+
+# Get firecracker
+go get -d ${firecracker_repo} || true
+# Checkout to specific version
+pushd "${GOPATH}/src/${firecracker_repo}"
+git checkout tags/${firecracker_version}
+./tools/devtool --unattended build --release -- --features vsock
```

Comment:
good, I had the idea you'd reuse the script on https://github.com/kata-containers/packaging/blob/master/static-build/firecracker/build-static-firecracker.sh to not duplicated code, but if this wont change a lot lets keep in this way. 
@mcastelino  @jodh-intel  any precerence here?